poetry.lock

```diff
@@ -523,7 +451,10 @@ files = [
 ]
 
 [package.dependencies]
-numpy = {version = ">=1.16,<2.0", markers = "python_version <= \"3.11\""}
+numpy = [
+    {version = ">=1.16,<2.0", markers = "python_version <= \"3.11\""},
+    {version = ">=1.26.0rc1,<2.0", markers = "python_version >= \"3.12\""},
+]
```

Comment:
If these conditional dependencies indeed work, doesn't that mean that we could also support Python 3.8 in the same way?

Probably not worth it to add Python 3.8 support back though.

Vice versa, in ScopeSim I just went directly for 1.26.3 for all Python versions, we could do that here too. But not worth it if this works.

Reply:
Huh, interesting, didn't notice that's in there. Maybe in the future we'll figure out how to properly do this. Certainly won't be the last time something like that occurs (probably as soon as python 3.13 drops)...